### ISSUE TITLE
Add Kotlin

### ISSUE THREAD

**Issue by niklas-heer** [niklas-heer/speed-comparison#82]
I'm searching for someone to implement a Kotlin version.

**Comment by Sanskruti-3** [comment]
Can you assign this to me?

**Comment by niklas-heer** [comment]
@Sanskruti-3 of course. I appreciate that you want to take this on and contribute. 🙏

**Comment by kaulex99** [comment]
@Sanskruti-3 Have you already started with the implementation?
